foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -334,6 +334,14 @@ def requeue_downloader_job(last_job: DownloaderJob) -> None:
         last_job.save()
         return
 
+    first_sample = original_file.samples.first()
```

Comment:
Can we setup for this query earlier by using `prefetch_related` when we retrieve the original_file?